prisma/schema.prisma

```diff
@@ -197,12 +197,12 @@ model lecture {
   description           String?
   appointmentType       lecture_appointmenttype_enum @default(legacy)
   isCanceled            Boolean?                     @default(false)
+  organizerIds          String[]                     @default([])
+  participantIds        String[]                     @default([])
```

Comment:
(I observed the same locally, probably some change in sort order of Prisma's db pull ... can stay)